### DISCUSSION TITLE
My Revolt account still logged. No access to 2FA. What should I do?

### DISCUSSION THREAD

**Discussion by goliv04053** [stoatchat/discussions#518] [Q&A]
Please, help. Or should I delete my account and make a new one? How should I do that?

**Comment by Fuzuki785** [comment]
If you can't log into your account because you lost your second factor of authentication **and your backup codes**, then you're out of luck really…

**Reply by goliv04053** [reply]
I'm logged into my account

**Reply by goliv04053** [reply]
The thing is.... I can't disable my 2FA without the codes

**Reply by Fuzuki785** [reply]
Then don't log out until you can recover access to your second factor or backup codes.
If they're definitively lost, then idk.

**Reply by Rexogamer** [reply]
Contact us at contact@revolt.chat

**Reply by goliv04053** [reply]
I sent a email right now

**Reply by goliv04053** [reply]
"To provide additional information, please reply to this email or click on the following link (for initial login, please request a new password)"
